presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -704,4 +704,34 @@ public void testTableStatsSystemTable()
         // cleanup
         assertUpdate("DROP TABLE test_table_stats");
     }
+
+    @Test
+    public void testAlterTable()
+            throws InterruptedException
+    {
+        assertUpdate("CREATE TABLE test_table (c1 bigint, c2 bigint)");
```

Comment:
Test methods can run concurrently, so table names need to be unique (or you get confusing sporadic failures). It's best to give them a name related to the test method, rather than something generic which might be used elsewhere.

How about naming it `alter_table`?